iroh/src/magicsock.rs

```diff
@@ -771,7 +793,9 @@ impl MagicSock {
             };
         }
 
+        #[cfg(not(wasm_browser))]
         let counter = self.poll_recv_counter.fetch_add(1, Ordering::Relaxed);
+        #[cfg(not(wasm_browser))]
```

Comment:
the two cfgs could be in a single block, making this a bit nicer

Reply:
Thanks for making me look at this again, doing the cfg on the whole function was much much better:
```rs
/// NOTE: Receiving on a [`Self::closed`] socket will return [`Poll::Pending`] indefinitely.
#[instrument(skip_all)]
#[cfg(not(wasm_browser))]
fn poll_recv(
    &self,
    cx: &mut Context,
    bufs: &mut [io::IoSliceMut<'_>],
    metas: &mut [quinn_udp::RecvMeta],
) -> Poll<io::Result<usize>> {
    // ...
}

/// poll_recv in browsers is "just" polling the relay receive path
#[cfg(wasm_browser)]
fn poll_recv(
    &self,
    cx: &mut Context,
    bufs: &mut [io::IoSliceMut<'_>],
    metas: &mut [quinn_udp::RecvMeta],
) -> Poll<io::Result<usize>> {
    self.poll_recv_relay(cx, bufs, metas)
}
```